tests/pipelines/pixart_alpha/test_pixart.py

```diff
@@ -324,6 +324,10 @@ def test_raises_warning_for_mask_feature(self):
     def test_inference_batch_single_identical(self):
         self._test_inference_batch_single_identical(expected_max_diff=1e-3)
 
+    # PixArt transformer model does not work with sequential offload so skip it for now
```

Comment:
I'm disable the sequential offloading test for now because it is not working correctly for transformer models 
created an issue about this https://github.com/huggingface/accelerate/issues/2701